README.md

```diff
@@ -319,7 +314,6 @@ Enable and configure experimental features.
 - `pubsub` (boolean): Enable libp2p pub-sub. (Default: `false`)
 - `ipnsPubsub` (boolean): Enable pub-sub on IPNS. (Default: `false`)
 - `sharding` (boolean): Enable directory sharding. Directories that have many child objects will be represented by multiple DAG nodes instead of just one. It can improve lookup performance when a directory has several thousand files or more. (Default: `false`)
-- `dht` (boolean): Enable KadDHT. **This is currently not interoperable with `go-ipfs`.**
```

Comment:
Will the DHT start automatically in the browser too?

Reply:
In the current state of this PR it is starting automatically in both browser and node. Do you think we should start by having only node.js nodes first?

Reply:
I would recommend to start with node only and look at adding it to the browser as a default later, if ever. Until we get more performance benchmarking for the browser I'm hesitant to just turn it on there for everyone.

Reply:
yeah, that's a good point!

Reply:
OK, so, I'm half inclined to just enable it in the browser by default, expose ourselves to the issues and get them fixed asap. That said, since this is such a big feature perhaps not having it enabled by default in the browser initially will give us some scope to test it out first and fix the bigger issues, so that we don't expose _all_ our users to the bugs and make everyone mad.

We do need a way to enable/disable it though - if the `dht` config option is being removed from `EXPERIMENTAL`, is it being added to the top level?